lib/ZwaveClient.js

```diff
@@ -867,9 +867,9 @@ function initNode (zwaveNode) {
   const node = this.nodes.get(nodeId)
 
   const deviceConfig = zwaveNode.deviceConfig || {
-    label: 'Unknown product ' + zwaveNode.productId,
-    description: zwaveNode.productType,
-    manufacturer: 'Unknown manufacturer ' + zwaveNode.manufacturerId
+    label: 'Unknown product ' + utils.num2hex(zwaveNode.productId),
+    description: utils.num2hex(zwaveNode.productType),
+    manufacturer: 'Unknown manufacturer ' + utils.num2hex(zwaveNode.manufacturerId)
   }
```

Comment:
I already evaluate this values later [here]. I suggest you to move that at the start of initNode function and use the values from the array. Also remember to use `zwaveNnode` instead of `node`